### PR TITLE
Workaround tj actions limitations with using newline as separator.

### PR DESCRIPTION
**Pull request by bhavanijayakumaran**
This will address

https://github.com/Cray/chapel-private/issues/3995

- [x]  Fix the body to have linefeeds between files rather than commas

Signed-off-by: bhavanijayakumaran <82669529+bhavanijayakumaran@users.noreply.github.com>

### CODE REVIEW

**Comment by bhavanijayakumaran** [comment]
This workflow will be only executed when the PR is merged.
I tested this fix in a separate branch.

Verified the changes using this draft PR :
https://github.com/chapel-lang/chapel/pull/21093/files

https://github.com/bhavanijayakumaran/chapel/actions/runs/3584763692/jobs/6031857537
   .github/workflows/CI.yml<br>.github/workflows/build-CI-container.yml <br>
   
The changed files are seperated by line feeds in the email instead of comma.
  
.github/workflows/CI.yml
.github/workflows/build-CI-container.yml

File: .github/workflows/email.yml [hunk 1]
```diff
@@ -42,7 +42,16 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v34
         with:
-          separator: ","                 
+          separator: "," 
+      # tj-actions' limiation prevents the user from using "\n" as ouptput seperator.
```
Comment (mstrout) [inline .github/workflows/email.yml:46]: "limitation" and "output" are misspelled

File: .github/workflows/email.yml [hunk 2]
```diff
@@ -42,7 +42,16 @@ jobs:
...
+      # parse the list of changed files and add <br> to add line feeds.
```
Comment (mstrout) [inline .github/workflows/email.yml:47]: Want to indicate this is the solution and indicate where you are adding <br>s.  "Solution: Parse the list of changed files and replace each comma delimiter with a <br>."

**Review by mstrout — CHANGES_REQUESTED** [review]
Here are some suggestions for the comments.  Let's also share screen so I can see how you are testing this.

File: .github/workflows/email.yml (hunk 1 above)
Comment (bhavanijayakumaran) [reply .github/workflows/email.yml]: Fixed

File: .github/workflows/email.yml (hunk 2 above)
Comment (bhavanijayakumaran) [reply .github/workflows/email.yml]: fixed.

**Review by mstrout — APPROVED** [review]
Looks good.

**Pull request merged**